transfer-hbar/script-transfer-hbar.js

```diff
@@ -63,8 +63,8 @@ async function scriptTransferHbar() {
     logger.log('The transfer transaction ID:', transferTxId.toString());
 
     // Sign the transaction with the account that is being debited (operator account) and the transaction fee payer account (operator account)
-    // Since the account that is being debited and the account that is paying for the transaction are the same only one accoun'ts signature is required
-    const transferTxSigned = await transferTx.sign(operatorKey);
+    // Since the account that is being debited and the account that is paying for the transaction are the same, only one account's signature is required
+for transferTxSigned = await transferTx.sign(operatorKey);
```

Comment:
replacing `const` with `for` here is likely to break the script. have you tried running it?

Reply:
oops, not sure why that is there. should just be `const`

Reply:
updated